docs/source/contributors.rst

```diff
@@ -0,0 +1,20 @@
+Contributions Welcome!
```

Comment:
I think we need a contribution guideline page.

Reply:
I have added the intial draft of community guideline page in the amended commit